### ISSUE TITLE
Performance issue in 7.1.0

### ISSUE THREAD

**Issue by malua** [Mawi137/ngx-image-cropper#592]
Hey,

first of all thanks for your hard work on the cropper!

I think in 7.1.0 there is an performance issue on mobile devices introduced.
With 7.0.3 the overlay responds instantly, in 7.1.0 it responds with multiple seconds delay.

I am using the cropper with autoCrop=false, so it's not connected to the cropping itself.

```
 <image-cropper
      [imageURL]="imageUrl"
      [maintainAspectRatio]="true"
      [aspectRatio]="1 / 1"
      format="png"
      [roundCropper]="true"
      [autoCrop]="false"
      (loadImageFailed)="loadImageFailed()"
    ></image-cropper>
```

Thank you for looking into it!

**Comment by Mawi137** [comment]
Hi
Are you using Ionic or Angular?

**Comment by jongbonga** [comment]
I also had an issue whereby pinching and moving the cropper area bounce around

**Comment by malua** [comment]
Ah sorry for the late reply. 

yes I am using Angular 16 + Ionic 7. I will try to make some videos tomorrow.

**Comment by MathanM** [comment]
I am also facing the same issue

**Comment by Mawi137** [comment]
Should be fixed in version `7.1.1`.
